Web_Development/Javascript/Topics/3.Web_Browser/README.md

```diff
@@ -1 +1,82 @@
-# To Do
\ No newline at end of file
+# Web Browser In Javascript
+
+**Browser** is the fundamental host environment for the JavaScript. It is a platform that extends the core JavaScript and provides additional features through specific objects and functions.
+</br>
+* The core object for **JavaScript** in web browser is the window object. It represents web browser window and simultaneously the ECMAScript Global object. 
+
+* The window object has properties that specify details of the browser window **(navigator, location, history) and methods (alert(), setTimeot())** which perform various operations.
+
+* Another **important part of JavaScript in web browser** is handling events. Event handler allow you to define function, that is called when specified event (for example an click) occurs.
+
+* The most important of them is **XMLHttpRequest object** which enables communication with servers.
+
+## **In this tutorial, we will see How to write JavaScript and what are the methods to write it.**
+
+First give the title(of your choice) as "Web Browser in JavaScript Tutorial" under the \<title> tag.
```

Comment:
This can be little confusing, you said to include title but given the full html scaffold at the last, its good to be placed here